examples/UPS/UPS.ino

```diff
@@ -91,12 +91,8 @@ void setup() {
 }
 
 void loop() {
-  
-  
   //*********** Measurements Unit ****************************
   bool bCharging = digitalRead(4);
-  bool bACPresent = bCharging;    // TODO - replace with sensor
```

Comment:
Presence of Ac is not the same as charging. You may have AC mains on but the charging may be off because your battery is fully charged.

Reply:
That's probably correct. My thinking was just to simplify your sample, so that it becomes slightly easier to understand for newbies like me. I have no problem keeping it if you prefer that.

Would you also prefer to keep the `bDischarging` variable, or can I slim down the PR to only remove that onw?

Reply:
It is better to keep as is because these variables are representing different flags in the bitmap status field as per USB PD specs.

Reply:
The fact that the underlying HID protocol treats `PRESENTSTATUS_ACPRESENT`, `PRESENTSTATUS_CHARGING` & `PRESENTSTATUS_DISCHARGING` separately is still clear by looking at the `UPS.ino` sources. I've just updated the PR with comments to make the "assume charging implies AC present" and "assume not charging implies discharging" assumptions explicit.

Still, I'm fine with you closing the PR if you're not interested in such changes.